lib/stream_rails/enrich.rb

```diff
@@ -29,7 +29,7 @@ def track_not_enriched_field(field, value = nil)
 
   class Enrich
     def initialize(fields = nil)
-      @fields = fields || [:actor, :object]
+      @fields = fields || [:actor, :object, :target]
```

Comment:
@kennym a bit OT in other libraries we auto detect references to enrich, can you create an issue for that?